### PR TITLE
aezeed/errors: add missing space in error message

### PR DESCRIPTION
**Pull request by parth-choudhary**

### CODE REVIEW

File: aezeed/errors.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ var (
 	// the specified mnemonic doesn't match. This indicates the user input
 	// the wrong mnemonic.
 	ErrIncorrectMnemonic = fmt.Errorf("mnemonic phrase checksum doesn't" +
-		"match")
+		" match")
```
Comment (halseth) [inline aezeed/errors.go]: space should be on previous line.

**Comment by parth-choudhary** [comment]
Made the asked/suggested changes.

**Comment by halseth** [comment]
@parth-choudhary Should also be squashed into one single commit, with the title 

```
aezeed/errors: add missing space in error message
```

**Comment by parth-choudhary** [comment]
Squashed :)

**Pull request merged**